jobs/config.json

```diff
@@ -8509,6 +8509,44 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-1-7-on-1-8": {
+    "args": [
+      "--cluster=",
+      "--deployment=kubernetes-anywhere",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-kubeadm-gce-1-7-on-1-8.env",
```

Comment:
the env files does not exist yet :-)

Reply:
oh, yeah forgot to do that before pushing. No wonder it fails :)

Reply:
oh those are all empty files :-( let's just don't include those lines

Reply:
May I clean these up in an upcoming PR?
I'd need to do the same for all other kubeadm jobs basically...
That way this PR is clean-ish :)